src/components/PokeTile.js

```diff
@@ -0,0 +1,37 @@
+import React, {useEffect, useState} from 'react';
+import './PokeTile.css';
+import apiRequest from "../RestApiRequest";
+
+function PokeTile( {pokemon, data} ) {
+
+    const [pokeData, setPokeData] = useState([]);
+
+    useEffect( () => {
+        const source = apiRequest( pokemon.url, setPokeData );
+
+        return function cleanup() {
+            source.cancel();
+        }
+    }, [data]);
+
+    // ?? Het lijkt altijd eerst 20 lege arrays te loggen en dan pas de gevulde ??
+    console.log( pokeData );
+
+    // Vandaar dat ik deze check er nog in heb gemaakt
+    return pokeData.length === 0 ? (
+        <></>
+    ) : (
+        <>
+            <ul className="pokemon-tile">
+                <section >
+                    <h1>{pokeData.name}</h1>
+                    <img src={pokeData.sprites.front_default} alt={pokeData.name}/>
+                    <h2>Moves: {pokeData.moves.length}</h2>
+                    <h2>Weight: {pokeData.weight}</h2>
+                </section>
+            </ul>
+        </>
+    );
```

Comment:
Heel goed dat je de length check hebt toegevoed, want dat is ook the way to go. Ik zou het alleen net iets anders doen, want dat leest iets makkelijker:
```suggestion
    if (pokeData.length === 0) return null;
    
    return (
        <>
            <ul className="pokemon-tile">
                <section >
                    <h1>{pokeData.name}</h1>
                    <img src={pokeData.sprites.front_default} alt={pokeData.name}/>
                    <h2>Moves: {pokeData.moves.length}</h2>
                    <h2>Weight: {pokeData.weight}</h2>
                </section>
            </ul>
        </>
    );
```